### ISSUE TITLE
Example with Nuxt module

### ISSUE THREAD

**Issue by smares** [histoire-dev/histoire#486]
### Clear and concise description of the problem

As a Nuxt module developer, I would like to see how to integrate Histoire so that I can view my custom components in a nice environment.

### Suggested solution

Add an example based on the [Nuxt Module Author Guide](https://nuxt.com/docs/guide/going-further/modules) with a sample component in `src/runtime/components/` that integrates Histoire and showcases the sample component.

### Validations

- [X] Follow our [Code of Conduct](https://github.com/histoire-dev/histoire/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/histoire-dev/histoire/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://histoire.dev/guide/).
- [X] Check that there isn't [already an issue](https://github.com/histoire-dev/histoire/issues) that requests the same feature to avoid creating a duplicate.

**Comment by lateek35** [comment]
@smares Did you find a way to make histoire works inside a Nuxt module ?

**Comment by smares** [comment]
> @smares Did you find a way to make histoire works inside a Nuxt module ?

Nope, I gave up on using Nuxt and Histoire due to various reasons.
